src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApproveConsentOrderContestedAboutToStartHandlerTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.IdamService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.OnStartDefaultValueService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONSENTED_ORDER_DIRECTION_DATE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONSENTED_ORDER_DIRECTION_JUDGE_NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_DATE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.CONTESTED_ORDER_APPROVED_JUDGE_NAME;
+
+public class ApproveConsentOrderContestedAboutToStartHandlerTest {
+
+    public static final String AUTH_TOKEN = "tokien:)";
+    private ApprovedConsentOrderContestedAboutToStartHandler handler;
+
+    @Mock
+    private IdamService service;
+
+    @Before
+    public void setup() {
+        handler =  new ApprovedConsentOrderContestedAboutToStartHandler(new OnStartDefaultValueService());
```

Comment:
Remove this setup assignment - you'll also want to mock the OnStartDefaultValueService

Reply:
Was unsure about this as the reference i'm using is the issue application default work that @rishikrsharma had done and he had used this. Happy to mock it however.